src/entries/SidePanel/SidePanel.tsx

```diff
@@ -420,8 +462,106 @@ function StepContent(
             </div>
           </div>
         )}
+        {inputs && inputs.length > 0 && !completed && (
+          <div className="flex flex-col gap-3 mt-3">
+            {inputs.map((input) => (
+              <InputField
+                key={input.name}
+                config={input}
+                value={inputValues[input.name] || ''}
+                onChange={(value) =>
+                  setInputValues((prev) => ({ ...prev, [input.name]: value }))
+                }
+                disabled={pending || notaryRequest?.status !== ''}
```

Comment:
IIRC I proposed this change, but it breaks it for me.
Why not just?
```suggestion
                disabled={pending}
```
or even remove it? This field probably does not need to be disabled if the step already is finished